packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -172,6 +179,11 @@ export class SequenceInterval implements ISerializableInterval {
         return result;
     }
 
+    public getIntervalId() {
```

Comment:
would be nice to add typing to this return as a string, to reduce anys from floating around